_config.yml

```diff
@@ -4,23 +4,20 @@ url: https://cs291.com # the base hostname & protocol for your site
 
 # User settings
 course_number: CS291A
-quarter: Fall 2022
+quarter: Fall 2023
 title: Scalable Internet Services
 
 github_username: scalableinternetservices
 
-instructor: Dr Nevena Golubovic
-instructor_email: nevena@cs.ucsb.edu
-instructor_url: https://www.linkedin.com/in/nevenagolubovic/
+instructor: Dr. Shyr-Shea Chang
+instructor_email: shyr-shea.chang@appfolio.com
+instructor_url: https://www.linkedin.com/in/shyr-shea-chang-6426271b8/
 
 drive_url: https://drive.google.com/drive/folders/11ZU37_a1Yc6pTBNo9uIcsCKV422Ypdyf?usp=sharing
 piazza_url: https://piazza.com/ucsb/fall2022/cs291a
```

Comment:
We need to update those too. Create a new piazza class and update the link here. Same for Google drive (making sure we have access to it)

Reply:
Done!